tests/docker-compose.yml

```diff
@@ -3,7 +3,7 @@
 version: "3.5"
 services:
   alpha1:
-    image: dgraph/dgraph:latest
+    image: dgraph/dgraph:${DGRAPH_IMAGE_TAG:-latest}
```

Comment:
python developers shouldn't have to build dgraph images locally - for them, it is sufficient to use latest dgraph images.  However for CI we want more granular control over which tag to use.